### PR TITLE
Added the Helmet meta to SeekApp

### PR DESCRIPTION
**Pull request by tamoore**
- Also added accessibility component for the `ScreenReaderOnly` h1 (both accessibility and seo implications I imagine).

### CODE REVIEW

File: react/SeekApp/SeekApp.js [hunk 1]
```diff
@@ -2,23 +2,46 @@ import styles from './SeekApp.less';
...
+import Helmet from 'react-helmet';
+import ScreenReaderOnly from 'Accessibility/ScreenReaderOnly';
 
-export default function SeekApp({ fullScreen, children }) {
+const defaultPageTitleAU = 'SEEK - Australia\'s no. 1 jobs, employment, career and recruitment site';
+const defaultPageTitleNZ = 'Jobs on SEEK - New Zealand\'s no. 1 Employment, Career and Recruitment site';
+
+const getLocalisedPageTitle = locale => locale === 'AU' ? defaultPageTitleAU : defaultPageTitleNZ;
+
+export default function SeekApp({ fullScreen, children, meta, link, title, locale }) {
```
Comment (tamoore) [inline react/SeekApp/SeekApp.js]: This could just be an object for `Helmet` but I thought making these explicit might be better?

File: package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     "less-loader": "2.2.3",
     "postcss-loader": "0.9.1",
     "raw-loader": "0.5.1",
+    "react-helmet": "3.1.0",
```
Comment (mjt01) [inline package.json]: This should be listed as a `peerDependency` for consuming apps to have installed. Also anything required for the demo app (this is) should be a `devDependency` not a `dependency`. Only thing in `dependencies` should be required webpack plugins for the config that gets decorated.

File: react/SeekApp/SeekApp.js [hunk 3]
```diff
@@ -2,23 +2,46 @@ import styles from './SeekApp.less';
...
+
   return (
     <div className={className}>
+      <Helmet
+        title={pageTitle}
+        meta={meta}
+        link={link}
+      />
+
+      <ScreenReaderOnly>
```
Comment (mjt01) [inline react/SeekApp/SeekApp.js]: If we hoist this up to the style guide is should probably be top level in the `react` directory and exported as part of the style guide package. Consuming apps should then refactor away their internal usage of `<ScreenReaderOnly />` in favour of this. We'll cover this in the release notes

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "seek-style-guide",
-  "version": "11.2.0",
+  "version": "11.3.0",
```
Comment (mjt01) [inline package.json]: As with the other modules, let's keep the version bump as a separate PR once this functionality is merged

File: react/SeekApp/SeekApp.js [hunk 5]
```diff
@@ -2,23 +2,46 @@ import styles from './SeekApp.less';
...
+import Helmet from 'react-helmet';
+import ScreenReaderOnly from 'Accessibility/ScreenReaderOnly';
 
-export default function SeekApp({ fullScreen, children }) {
+const defaultPageTitleAU = 'SEEK - Australia\'s no. 1 jobs, employment, career and recruitment site';
+const defaultPageTitleNZ = 'Jobs on SEEK - New Zealand\'s no. 1 Employment, Career and Recruitment site';
+
+const getLocalisedPageTitle = locale => locale === 'AU' ? defaultPageTitleAU : defaultPageTitleNZ;
+
+export default function SeekApp({ fullScreen, children, meta = [], link = [], title, locale = 'AU' }) {
```
Comment (mjt01) [inline react/SeekApp/SeekApp.js]: I think it's cleaner to move these defaults to `defaultProps`, w're already doing this for `fullScreen`

**Comment by mjt01** [comment]
lgtm :art:
